### PR TITLE
Version Bump to 0.23.0

### PR DESCRIPTION
**Pull request by vsbharath**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
     name="sfdo-template-helpers",  # Required
     # https://www.python.org/dev/peps/pep-0440/
     # https://packaging.python.org/en/latest/single_source_version.html
-    version="0.20.0",  # Required
+    version="0.22.0",  # Required
```
Comment (jstvz) [inline setup.py:26]:
```suggestion
    version="0.23.0",  # Required
```

**Review by jstvz — CHANGES_REQUESTED** [review]
Needs to reflect the next tag, no?

**Comment by vsbharath** [comment]
Thought of deleting the existing tag and re pushing it. So that we don't
have to make any changes for MetaDeploy. If the app fails to pull latest we
can have new tag.

**Review by jstvz — APPROVED** [review]

**Comment by jstvz** [comment]
> Thought of deleting the existing tag and re pushing it. So that we don't have to make any changes for MetaDeploy. If the app fails to pull latest we can have new tag.

Fair enough. I have a bias towards thinking of tags as immutable when they're not.

**Pull request merged**